src/components/admin/AppLayout.vue

```diff
@@ -1,7 +1,13 @@
 <template>
```

Comment:
![image](https://user-images.githubusercontent.com/5394573/60192915-6bc5d680-983f-11e9-99c3-4e12cebb80dc.png)

Reply:
@asvae the better decision is to set only min-height (not height) of sidebar-link

Reply:
@nastassiadanilova I said nothing about min-height nor height.